src/shared-components/theme-engine/_theme.scss

```diff
@@ -52,3 +52,6 @@ $underlay-background: var(--underlay-background, rgba(0,0,0,0.95));
 $glow-text-color: var(--glow-text-color, hsla(0,0%,100%,0.7));
 $glow-background-color: var(--glow-background-color, #6e10bb);
 $glow-transition-background-color: var(--glow-transition-background-color, #e0baff);
+$glow-transition-background-color: var(--glow-transition-background-color, #e0baff);
```

Comment:
is this line actually duplicated, or just a merge artifact?

Reply:
nice catch, I've fixed it